app/api/routes_admin.py

```
@@ -0,0 +1,752 @@
+        raise_http_error(status.HTTP_400_BAD_REQUEST, "extract_failed", str(exc))
+    except Exception as exc:
+        try:
+            inc_error("extract_failed")
+        except Exception:
+            pass
+        raise_http_error(status.HTTP_500_INTERNAL_SERVER_ERROR, "extract_failed", str(exc))
+
+    # Validate using default runtime factory from runtime_manager
+    try:
+        from app.model.runtime_manager import _runtime_factory as _rtf  # type: ignore
+    except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)